lightpath/controller.py

```diff
@@ -432,15 +465,8 @@ def make_graph(
         # label nodes with device name, store ophyd device
         nodes = []
         for res in result_list:
-            try:
-                dev = res.get()
-            except Exception:
-                # TODO: be better about specific exceptions
-                logger.debug(
-                    f'Failed to initialize device: {res["name"]}'
-                )
-                continue
-            nodes.append((res.metadata['name'], {'dev': dev}))
+            nodes.append((res.metadata['name'],
+                         {'res': res, 'dev': None}))
```

Comment:
So this is the core of the PR here? Store the happi result with each node and use the happi result whenever you want to check the metadata instead of needing to build all the devices. I think this is a good choice and it looks like you propagated it everywhere.

Reply:
Yep, you got it.  As usual a bunch of other things crept in before I told myself to stop